VAGRANT.md

```diff
@@ -43,15 +43,16 @@ is.
     
     See the FAQ how to skip this step and point Nominatim to an existing database.
 
-  ```
-  # inside the virtual machine:
-  mkdir data
-  cd build
-    wget --no-verbose --output-document=../data/monaco.osm.pbf http://download.geofabrik.de/europe/monaco-latest.osm.pbf
-    ./utils/setup.php --osm-file ../data/monaco.osm.pbf --osm2pgsql-cache 1000 --all 2>&1 | tee monaco.$$.log
-    ```
-
-  To repeat an import you'd need to delete the database first
+      ```
+      # inside the virtual machine:
+      mkdir data
+      cd build
+      wget --no-verbose --output-document=../data/monaco.osm.pbf http://download.geofabrik.de/europe/monaco-latest.osm.pbf
+      wget --no-verbose --output-document=../Nominatim/data/country_osm_grid.sql.gz http://www.nominatim.org/data/country_grid.sql.gz
```

Comment:
Oh, I see. Sorry about the confusion! 

The country grid file is necessary to start the import. So, I just thought it would be better to document it here too.